code/modules/mob/living/simple_animal/friendly/gondola.dm

```diff
@@ -1,26 +1,65 @@
+#define GONDOLA_HEIGHT_LONG "gondola_body_long"
+#define GONDOLA_HEIGHT_AVERAGE "gondola_body_medium"
+#define GONDOLA_HEIGHT_SHORT "gondola_body_short"
+
+#define GONDOLA_COLOR_LIGHT "A87855"
+#define GONDOLA_COLOR_AVERAGE "915E48"
+#define GONDOLA_COLOR_DARK "683E2C"
+
+#define GONDOLA_MOUSTACHE_LARGE "gondola_moustache_large"
+#define GONDOLA_MOUSTACHE_SMALL "gondola_moustache_small"
+
+#define GONDOLA_EYES_CLOSE "gondola_eyes_close"
+#define GONDOLA_EYES_FAR "gondola_eyes_far"
+
 //Gondolas
 
 /mob/living/simple_animal/pet/gondola
 	name = "gondola"
 	real_name = "gondola"
 	desc = "Gondola is the silent walker. Having no hands he embodies the Taoist principle of wu-wei (non-action) while his smiling facial expression shows his utter and complete acceptance of the world as it is. Its hide is extremely valuable."
-	response_help  = "pets"
+	response_help = "pets"
 	response_disarm = "bops"
-	response_harm   = "kicks"
-	emote_see = list("watches.", "stares off into the distance.","contemplates.")
+	response_harm = "kicks"
 	faction = list("gondola")
 	turns_per_move = 10
 	icon = 'icons/mob/gondolas.dmi'
-	icon_state = "gondola"
-	icon_living = "gondola"
-	icon_dead = "gondola_dead"
-	butcher_results = list(/obj/item/stack/sheet/animalhide/gondola = 1)
+	icon_state = null
+	icon_living = null
+	loot = list(/obj/effect/decal/cleanable/blood/gibs, /obj/item/stack/sheet/animalhide/gondola = 1)
 	//Gondolas aren't affected by cold.
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	maxbodytemp = 1500
 	maxHealth = 200
 	health = 200
+	del_on_death = TRUE
```

Comment:
Why are you gibbing gondolas on death? Are there issues with rotating them?

Reply:
if it's rotated then revived it will stay rotated, i tried to fix this by adding a dead eyes overlay to the gondola's sprite, rotating that then setting that to icon_dead but i gave up after that failed.

Reply:
Ok